contracts/proxy-lib/src/mutate.rs

```diff
@@ -155,7 +155,7 @@ impl ProxyContract {
                 } => {
                     let account_id: AccountId =
                         AccountId::from_str(receiver_id.as_str()).expect("Invalid account ID");
-                    Promise::new(account_id).transfer(NearToken::from_near(amount))
+                    Promise::new(account_id).transfer(NearToken::from_yoctonear(amount))
```

Comment:
Good catch, this could've been really bad. 🎉 

Also would've prevented sending fractions of a near.

And made some random guy really really rich 😅

Reply:
:)